### ISSUE TITLE
Unwanted CSS output when using scoped skin 

### ISSUE THREAD

**Issue by CestDiego** [eBay/skin#142]
We have a problem with the way less processes our scoped solution.

if we do

```less
// dialog.less
.dialog-mixin() {

.dialog {
    &--show-init {
        // for specificity
        && {
            display: block;
        }
    }
}
```

```less
.ds6 {
   .dialog-mixin()
}
```

This will transform it into:

```less
.ds6 .dialog--show-init.ds6 .dialog--show-init  {
   display: block;
}
```

Which doesn't match anything. 

A solution would be to instead of requiring the `less` files, we include the `css` parsed files under `dist`

**Comment by DylanPiercey** [comment]
@seangates this is what I was talking about this morning. Looks like @CestDiego beat me to creating and issue 😄.

**Comment by seangates** [comment]
@DylanPiercey I asked him to. 👍 😄

**Comment by CestDiego** [comment]
This is fixed by #143
